packages/react/src/patterns/sections/CardSection/__stories__/CardSection.stories.js

```diff
@@ -34,12 +34,19 @@ storiesOf('Patterns (Sections)|CardSection', module)
 
     cards[type] = data;
 
+    const cta = {
+      heading: 'Top level card link',
+      cta: {
+        href: 'https://www.example.com',
+      },
+    };
+
     return (
       <CardSection
         heading={text('Heading (required)', cardsTitle)}
-        cards={cards[type]}
-        data={data}
         theme={theme}
+        cards={data}
+        cta={type === 'CardSectionSimple' ? cta : null}
```

Comment:
Is it worth adding a way to toggle this in the story since it's optional?

Reply:
I added because functional spec says we need to show the CTA inverse only in CardSectionSimple pattern.

Reply:
Not sure you understood my comment. We don't have a way to toggle the CTA on and off. For cardsection I'm not sure it's necessary to be specific, but that's ok too.